lib_web/context.ml

```diff
@@ -72,15 +80,19 @@ let template t contents =
   let open Tyxml.Html in
   html_to_string (
     html
-      (head (title (txt site.name)) [
+      (head (title (txt site.name))
+        ((List.map (fun font -> link ~rel:[ `Stylesheet ] ~href:font ()) fonts)
+        @ [
           link ~rel:[ `Stylesheet ] ~href:"/css/style.css" ();
-          meta ~a:[a_charset "UTF-8"] ();
-        ]
+          link ~rel:[ `Icon ] ~href:img_ocaml_logo ();
+          meta ~a:[ a_charset "UTF-8" ] ();
+        ])
       )
       (body [
           nav [
+            img ~alt:"" ~src:img_ocaml_logo ~a:[ a_height 29 ] ();
+            div [code [txt site.name]];
```

Comment:
The example apps do use the shell command as their name, but a pipeline's name is not "code" in general. Maybe just put a class on this and let the CSS decide what to do?

Reply:
I've changed this to a CSS class.